test/test_metrics.py

```diff
@@ -221,6 +221,15 @@ def test_perfect_fit(self, device, dtype):
         actual = kornia.metrics.aepe(sample, sample, reduction="none")
         assert_close(actual, expected)
 
+    def test_aepe_alias(self, device, dtype):
+        sample = torch.ones(4, 4, 2, device=device, dtype=dtype)
+        expected = torch.zeros(4, 4, device=device, dtype=dtype)
+        actual_aepe = kornia.metrics.aepe(sample, sample, reduction="none")
+        actual_alias = kornia.metrics.endpoint_error.average_endpoint_error(sample, sample, reduction="none")
```

Comment:
can you expose it on kornia.metrics.__init__.py ? - to use as `kornia.metrics.average_endpoint_error` too